modules/apps/27-interchain-accounts/types/codec_test.go

```diff
@@ -0,0 +1,130 @@
+package types_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/cosmos/ibc-go/v2/modules/apps/27-interchain-accounts/types"
+	"github.com/cosmos/ibc-go/v2/testing/simapp"
+)
+
+// caseRawBytes defines a helper struct, used for testing codec operations
+type caseRawBytes struct {
+	name    string
+	bz      []byte
+	expPass bool
+}
+
+// mockSdkMsg defines a mock struct for testing codec error scenarios
```

Comment:
Make godocs consistent 

```suggestion
// mockSdkMsg defines a mock struct, used for testing codec error scenarios
```